leveldb-benchmark/src/main/java/org/iq80/leveldb/benchmark/DbBenchmark.java

```diff
@@ -484,16 +484,18 @@ private void readReverse(ThreadState thread)
 
     private void readRandom(ThreadState thread)
     {
-        int found = 0;
+
+        long bytes = 0;
         for (int i = 0; i < reads; i++) {
             byte[] key = formatNumber(thread.rand.nextInt(num));
             byte[] value = db.get(key);
-            if (value != null) {
-                found++;
+            if (value == null) {
+                throw new NullPointerException(String.format("db.get(%s) is null", new String(key, UTF_8)));
```

Comment:
You can't assume DB has data.
We must not fail when key is not found has concurrent write may be running.